spec.html

```diff
@@ -36297,16 +36297,24 @@ <h1>
           <h1>
             GetStringIndex (
               _S_: a String,
-              _e_: a non-negative integer,
+              _cpIndex_: a non-negative integer,
             ): a non-negative integer
           </h1>
           <dl class="header">
+            <dt>description</dt>
+            <dd>It interprets _S_ as a sequence of UTF-16 encoded code points, as described in <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and returns the code unit index corresponding to code point index _cpIndex_, clamped to a maximum position just beyond the end of _S_.</dd>
```

Comment:
```suggestion
            <dd>It interprets _S_ as a sequence of UTF-16 encoded code points, as described in <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and returns the code unit index corresponding to code point index _cpIndex_ when such an index exists. Otherwise, the length of _S_ is returned.</dd>
```

Reply:
Done.